src/alignment.jl

```diff
@@ -79,6 +79,10 @@ function Alignment(cigar::AbstractString, seqpos::Int=1, refpos::Int=1)
                 seqpos += n
             elseif isdeleteop(op)
                 refpos += n
+            elseif ismetaop(op)
+                # Meta operations consume alignment positions, but not sequence or reference
+                # positions, so there is nothing to do here but prevent the "not supported"
+                # error
```

Comment:
```suggestion
                # error
                nothing
```

Merely a suggestion, but I'd prefer to be explicit that nothing is happening :shrug: - probably pedantic, but an empty conditional just doesn't sit right.

There's probably a better way to do this with a Holy trait or something, but this is fine.